src/map/js/config.js

```diff
@@ -336,7 +336,7 @@ export const config = {
       visible: true,
       layerIds: [21],
       className: 'active-fires',
-      url: 'https://gfw-staging.wri.org/arcgis/rest/services/Fires/FIRMS_Global_MODIS_24hrs/MapServer/',
+      url: 'https://gis-gfw.wri.org/arcgis/rest/services/Fires/FIRMS_Global_VIIRS_24hrs/MapServer',
```

Comment:
This should be a `url` to `https://gis-gfw.wri.org/arcgis/rest/services/Fires/FIRMS_Global_MODIS_24hrs/MapServer`, not it's VIIRS counterpart.

Reply:
Updated